ci/gpu/build.sh

```diff
@@ -79,21 +79,10 @@ gpuci_conda_retry build -c ${LIBCUCIM_BLD_PATH} -c conda-forge/label/cupy_rc -c
 gpuci_logger "Install cuCIM and its dependencies"
 
 gpuci_logger "Install dependencies"
-gpuci_conda_retry create -n cucim -y -c conda-forge -c conda-forge/label/cupy_rc -c rapidsai-nightly \
+gpuci_conda_retry install -y -c conda-forge -c rapidsai-nightly \
     rapids-doc-env \
```

Comment:
Not sure if we still need to explicitly install this or if it is already included